test/declaration_test.dart

```diff
@@ -774,6 +774,17 @@ div {
   expect(stylesheet != null, true);
   expect(errors.isEmpty, true, reason: errors.toString());
   expect(prettyPrint(stylesheet), generated3);
+
+  final input4 = '''
+div {
+  filter: FlipH;
+}''';
```

Comment:
Yes the issue occurs in isolation. The reason that filter property appears to work in the previous test is because there is a following filter with a functional expression. Essentially what's happening is when the parser finds a `filter` property, it parses an identifier, then just eats up anything until a `)`.

Here's an excerpt from the previous test case:
```css
div {
  ...
  filter: FlipV;
  filter: Gray;
  filter: Chrome(color = #000000) Mask(Color=#00FF00);
  ...
}
```
And here's the debug string of the AST:
```
    Expressions  (line 6, column 11: 
  Filter: FlipV;
          ^^^^^)
     expressions [
      LiteralTerm  (line 6, column 11: 
  Filter: FlipV;
          ^^^^^^^)
       value: FlipV;
  Filter: Gray;
  FILTER: Chroma(Color = #000000) 
      LiteralTerm  (line 8, column 35: 
  FILTER: Chroma(Color = #000000) Mask(Color=#00FF00);
                                  ^^^^^^^^^^^^^^^^^^^)
       value: Mask(Color=#00FF00)
     ]
```

As you can see, `expressions` should only contain a `LiteralTerm` with value `FlipV`, but instead it's everything until the first following `)`.